### PR TITLE
Adding Newsletter: Redesign + Diego Gómez

### PR DESCRIPTION
**Pull request by gedankenstuecke**
Also fixes a duplication in our footer in the `user_mailer.html.erb` layout.

### CODE REVIEW

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -97,7 +97,7 @@ def survey(user)
                          password: ENV.fetch('SURVEY_EMAIL_PASSWORD'),
                          address: ENV.fetch('SURVEY_EMAIL_ADDRESS'),
                          port: ENV.fetch('SURVEY_EMAIL_PORT') }
-    mail(subject: 'openSNP: Read our survey results and meet Open Humans',
+    mail(subject: 'openSNP: a fresh design and a plea for help',
```
Comment (tsujigiri) [inline app/mailers/user_mailer.rb]: Maybe with a capital A after the colon? :woman_teacher:

**Comment by tsujigiri** [comment]
We really need a nicer way to send these.

**Comment by gedankenstuecke** [comment]
It's already much nicer since we got the layout for the mailer but yep, it's not really fun like this. But what would be a good way to make it easier?

**Comment by tsujigiri** [comment]
It could be as simple as a form, the content of which we pass into [`simple_format`](http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-simple_format) and send it off.

**Comment by gedankenstuecke** [comment]
Good idea, for that we'd need some flag which users are allowed to send out newsletters I guess  😉

**Comment by tsujigiri** [comment]
I'd just hard-code a list somewhere. It's not like this changes very often.

**Comment by tsujigiri** [comment]
Or maybe an environment variable to avoid putting email addresses on here, where they are visible to spam bots.

**Comment by gedankenstuecke** [comment]
Yep, that sounds good to me. 😄

**Pull request merged**